### DISCUSSION TITLE
Sample/Documentation for VideoFrames in Compose

### DISCUSSION THREAD

**Discussion by karthik-cherryz** [coil-kt/coil#1559] [General]
Still searching for a proper documentation on how to use VideoFrames in compose. :( Please point me in the right direction.

**Comment by colinrtwhite** [comment]
Add the [`VideoFrameDecoder`](https://coil-kt.github.io/coil/videos/) to your [`ImageLoader`](https://coil-kt.github.io/coil/getting_started/#image-loaders) then you can load video frames like so:

```kotlin
AsyncImage(
    model = ImageRequest.Builder(LocalContext.current)
        .data(url)
        .videoFrameMillis(1000)
        .build(),
    contentDescription = null,
)
```
